VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentTrackerImpl.java

```diff
@@ -38,12 +40,14 @@ public ISegmentTrackerImpl(Context context, Config config) {
 
             if (writeKey != null) {
                 logger.debug("SegmentTracker created with write key: " + writeKey);
+                Analytics.LogLevel logLevel = Analytics.LogLevel.NONE;
+                if (debugging) logLevel = Analytics.LogLevel.VERBOSE;
```

Comment:
I think we can get rid of this variable, and initialize the value inside the builder:

````java
.logLevel(debugging ? Analytics.LogLevel.VERBOSE :
        Analytics.LogLevel.NONE)
````